homeassistant/components/nut/__init__.py

```diff
@@ -39,8 +39,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     # strip out the stale options CONF_RESOURCES
     if CONF_RESOURCES in entry.options:
+        new_data = entry.data.copy()
+        new_data[CONF_RESOURCES] = entry.options[CONF_RESOURCES]
```

Comment:
```suggestion
        new_data = {**entry.data, CONF_RESOURCES: entry.options[CONF_RESOURCES]}
```

Reply:
> For the future, I suggest we remove the resources selection from the config flow completely and just set a default selection from the beginning

Totally agree, it was already in my mind, I was just waiting for a conclusion of PR #58798 to have an idea of what should be enabled by default.